setup.cfg

```diff
@@ -3,18 +3,18 @@ current_version = 1.0.0
 commit = True
 tag = True
 
-[pylint]
-ignore = CVS
-max-line-length = 120
-
 [pycodestyle]
 count = False
-ignore = E501, W503
+ignore = E501, W503, R903
 max-line-length = 120
 statistics = True
 
 [flake8]
 max-line-length = 120
+ignore = R903
+per-file-ignores =
+    */__init__.py: F403
+	*/settings/*.py: F403
```

Comment:
I think you forgot to remove this

Reply:
403 is related to import `*` and that causes problem in flake8.
![image](https://user-images.githubusercontent.com/51926076/220456584-a22ef9fe-af27-4d3e-a1c6-0273fc58b842.png)